### PR TITLE
copy buttons are added in missing commands

### PR DESCRIPTION
**Pull request by gunjanmimo**
Signed-off-by: gunjanmimo <gunjan.mtbpaul@gmail.com>

### CODE REVIEW

**Comment by gunjanmimo** [comment]
fixes #106

**Comment by gunjanmimo** [comment]
Sir, I have added "Join the Community" button

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.6.5'
+ruby '2.6.3'
```
Comment (Aisuko) [inline Gemfile]: Please check the version. I believe `2.6.5` would make more sense.

Comment (gunjanmimo) [reply Gemfile]: Sir, I have made the change

File: _includes/partials/contributor.html [hunk 2]
```diff
@@ -1,62 +1,72 @@
...
-
-                        <div class="twitter"><img src="{% include relative-src.html src='../images/social/twitter.png'%}"> 
-                            <a target="_blank" href="https://twitter.com/{{ page.twitter }}">{{ page.twitter }}</a></div>
-
-                    {% endif %}
-                    {% if page.github %}
-                        
-                        <div class="github"><img src="{% include relative-src.html src='../images/social/github.png'%}"> 
-                            <a target="_blank" href="https://github.com/{{ page.github }}">{{ page.github }}</a></div>
+<!DOCTYPE html>
```
Comment (leecalcote) [inline _includes/partials/contributor.html]: The contributor.html file is intentionally a partial html document. This file should not have <html><head><body> tags in it. Those pages are added elsewhere as the site is built.

File: _includes/partials/contributor.html [hunk 3]
```diff
@@ -1,62 +1,72 @@
...
+    <head>
 
+    </head>
+    <body>
+        <div class="contributor flex"> 
+            <div class="container {% if page.full_width %}max-container{% endif %}">
+                <h2>Community Members</h2>
+                <p>A warm and welcoming collection of open sourcers.</p>
+                <h4 style="text-align:center;color:aliceblue;">
+                    <input style="padding-bottom:10px;padding-top:10px;width:300px; margin-left: auto; margin-right: auto;" type="submit" value="Join the Community" name="subscribe" id="mc-embedded-subscribe"></h4></span>
```
Comment (leecalcote) [inline _includes/partials/contributor.html]: The button visually behaves well, but does not link the user anywhere. This should navigate the user to the community Slack.

**Comment by gunjanmimo** [comment]
@leecalcote sir, is this pr correct now?

File: _includes/partials/contributor.html [hunk 4]
```diff
@@ -1,62 +1,72 @@
...
-            {% endif %}
-        {% endif %}
-        </ul>
-    </div>	
-</div>
\ No newline at end of file
+                </ul>
+            </div>	
+        </div>
+    </body>
```
Comment (leecalcote) [inline _includes/partials/contributor.html]: @gunjanmimo the <html> and <body> tags need to be removed.

File: _includes/getting-started.html [hunk 5]
```diff
@@ -63,44 +66,84 @@ <h3 style="margin-left:1em;">Kubernetes user</h3 style="margin-left:1.5em;">
...
+          <div style="width:auto; height:60px; padding:0px; position:relative;">
+            <pre><code style="top: 10px;position: relative;"> $ kubectl create ns meshery<br /> $ kubectl -n meshery apply -f deployment_yamls/k8s
+            </code></pre>
+            <!-- copy to clipboard -->
+            <a class="btn waves-effect waves-dark white-text darken-2 l5-light-blue z-depth-4"
+            style="position: absolute;top:-14px; right: -1px; border: 0; 
+                cursor: pointer; background:#00a4ca; width:60px; height:75px; margin-bottom:35px; 
+                border-radius: 2px; margin:0px 0px 880px 0px;"
+              data-clipboard-target="#start-brew"
+              data-clipboard-text="brew tap layer5io/tap
```
Comment (leecalcote) [inline _includes/getting-started.html]: @gunjanmimo you'll need to update the clipboard data for this section.

File: _includes/getting-started.html [hunk 6]
```diff
@@ -63,44 +66,84 @@ <h3 style="margin-left:1em;">Kubernetes user</h3 style="margin-left:1.5em;">
...
+            <div style="width:auto; height:60px; padding:0px; position:relative;">
+              <pre><code style="top: 15px;position: relative;"> $ git clone https://github.com/layer5io/meshery.git; cd meshery
+              </code></pre>
+              <!-- copy to clipboard -->
+              <a class="btn waves-effect waves-dark white-text darken-2 l5-light-blue z-depth-4"
+              style="position: absolute;top:-14px; right: -1px; border: 0; 
+                  cursor: pointer; background:#00a4ca; width:60px; height:75px; margin-bottom:35px; 
+                  border-radius: 2px; margin:0px 0px 880px 0px;"
+                data-clipboard-target="#start-brew"
+                data-clipboard-text="brew tap layer5io/tap
```
Comment (leecalcote) [inline _includes/getting-started.html]: Change clipboard data to be Kubernetes specific.

**Pull request closed without merging**